lib/core/service/event_pass_service/event_pass_service.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:io';
+
+import 'package:app/core/config.dart';
+import 'package:app/core/oauth/oauth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_wallet_card/core/passkit.dart';
+import 'package:http/http.dart';
+import 'package:flutter_wallet_card/flutter_wallet_card.dart';
+import 'package:path_provider/path_provider.dart';
+
+class EventPassService {
+  final Client _client = Client();
```

Comment:
`import 'package:http/http.dart' as http;`
`final http.Client _client = http.Client();` 

The reason is we have lots of client, adding package name as a prefix to make it straight forward